lib/mxcreply.cpp

```diff
@@ -64,9 +64,11 @@ MxcReply::MxcReply(QNetworkReply* reply, Room* room, const QString &eventId)
     auto eventIt = room->findInTimeline(eventId);
     if(eventIt != room->historyEdge()) {
         auto event = eventIt->viewAs<RoomMessageEvent>();
-        if (auto* efm = std::get_if<EncryptedFileMetadata>(
-                &event->content()->fileInfo()->source))
-            d->m_encryptedFile = *efm;
+        if (event) {
```

Comment:
```suggestion
        if (auto event = eventIt->viewAs<RoomMessageEvent>()) {
```
Too bad GitHub doesn't let me use unchanged lines in the suggestion but you get the idea.